numbat/modules/math/geometry.nbt

```diff
@@ -2,9 +2,11 @@ use core::functions
 use math::constants
 
 @description("The length of the hypotenuse of a right-angled triangle $\\sqrt\{x^2+y^2\}$.")
+@example("hypot2(3, 4)")
 fn hypot2<T: Dim>(x: T, y: T) -> T = sqrt(x^2 + y^2)
 
 @description("The Euclidean norm of a 3D vector $\\sqrt\{x^2+y^2+z^2\}$.")
+@example("hypot3(4, 1, 4)")
```

Comment:
We could use a [Pythagorean quadruple](https://en.wikipedia.org/wiki/Pythagorean_quadruple) as an example :smiley: 

```suggestion
@example("hypot3(8, 9, 12)")
```